scripts/lint.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+exit_code=0
+
+for f in `find ./kore -name '*.hs'`; do
```

Comment:
I think we have a more general invocation to find Haskell source files in one of the other scripts (`check.sh`?). Should we extract it for consistency?

Reply:
Thanks, done (it was `stylish.sh`).